client/src/components/IIITDMJ/iiitdmj.scss

```diff
@@ -0,0 +1,65 @@
+* {
+    box-sizing: border-box;
+  }
+  
+  body {
+    margin: 0;
+  }
+
+  .nav-menu{
+    /* margin-top: 50px; */
+
+  }
+  .nav-menu ul{
+    padding: 0;
+    margin: 0;
+    list-style: none;
+    display: flex;
+    gap: 4rem;
+
+  }
+    .nav-menu li{
+        padding: 1.5rem;
+        font-family: 'Montserrat';
+        font-style: normal;
+        font-weight: 500;
+        font-size: 22px;
+        line-height: 17px;
+        color: #EBEBEB;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+        border-bottom : 3.5px solid #40424c;
+    }
+
+    .nav-menu li:hover{
+        background-color: #2c2e38;
+    }
+    .nav-menu-title{
+        color: #EBEBEB
+    }
+    .iiitdmj-logo{
+        width: 100px;
+        height: 97.73px;
+    }
+    .iiitdmj-name{
+        font-family: 'Montserrat';
+        font-style: normal;
+        font-weight: 500;
+        font-size: 22px;
+        line-height: 17px;
+        color: #EBEBEB;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+        margin-left: 1rem;
+        margin-top: -90px;
```

Comment:
@rajutkarsh07 please fix the CSS of this file, so that I can merge this component! try to pull this branch to your local so to verify the correct CSS changes, and push the changes to the Pratik's forked repo, he will merge the changes and the changes will be reflected here also in this PR